### PR TITLE
#340/RocketAnimated

### LINKED ISSUE #340: UI | Desktop | Dynamicznie poruszająca się w stronę myszki grafika

### PR DESCRIPTION
**Pull request by pierwszazlewej**
closes #340 
- animation changed
obraz

### CODE REVIEW

File: client/src/Molecules/RocketWithPlanets/RocketWithPlanetsStyled.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import planets from '../../Assets/planets.png';
+import styled from 'styled-components';
+
+const RocketWithPlanetsStyled = styled.div`
+  @media only screen and (min-width: 1600px) {
```
Comment (vieraboschkova) [inline client/src/Molecules/RocketWithPlanets/RocketWithPlanetsStyled.js:5]: It shows up on smaller screens for me, you could add `  display: none;
` between lines 4 and five not to show on smaller than 1600. Check the display for 1601px, the rocket is not visible 
image

File: client/src/Molecules/RocketWithPlanets/RocketWithPlanetsStyled.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    display: block;
+    grid-area: 4 / 4 / 5 / 5;
+    background-image: url(${planets});
+    background-repeat: no-repeat;
+    width: 100%;
+    height: 100%;
+    z-index: 1;
+    position: relative;
+    bottom: 4.9375rem;
+    left: 9.375rem;
```
Comment (vieraboschkova) [inline client/src/Molecules/RocketWithPlanets/RocketWithPlanetsStyled.js:15]: These look like magic numbers :D

File: client/src/Molecules/RocketWithPlanets/RocketWithPlanetsStyled.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+const RocketWithPlanetsStyled = styled.div`
+  @media only screen and (min-width: 1600px) {
+    display: block;
+    grid-area: 4 / 4 / 5 / 5;
+    background-image: url(${planets});
+    background-repeat: no-repeat;
+    width: 100%;
+    height: 100%;
+    z-index: 1;
```
Comment (vieraboschkova) [inline client/src/Molecules/RocketWithPlanets/RocketWithPlanetsStyled.js:12]: Check if any of those you could get rid of, perhaps some are not necessary.

File: client/src/Molecules/RocketWithPlanets/RocketWithPlanetsStyled.js (hunk 2 above)
Comment (pierwszazlewej) [reply client/src/Molecules/RocketWithPlanets/RocketWithPlanetsStyled.js:15]: It' result of crazy calculations by 'px to rem' plugin, Should I change it? :)

**Review by Smyku6 — APPROVED** [review]
Real rocket science! Elon will be pleased!
image
Lajka approved

**Pull request merged**